network-api/networkapi/templates/fragments/buyersguide/primary_nav.html

```diff
@@ -13,8 +14,7 @@
 {% endblock %}
 
 {% block narrow_screen_nav_links %}
-{% url 'buyersguide-home' as home_url %}
-<div><a class="{% bg_active_nav request.get_full_path home_url %}" href="{% relocalized_url home_url %}">{% trans "Privacy Not Included" %}</a></div>
+<div><a class="{% bg_active_nav request.get_full_path home_page.url %}" href="{% relocalized_url home_page.url %}">{% trans "Privacy Not Included" %}</a></div>
```

Comment:
Updated to use `home_page.url` instead of `home_url`, which actually did not work anyways! 
Thankfully this was just used to decide whether or not to highlight the link, based on whether the user was on the page or not.